testdata/default/cheats/ExpectCreate.t.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.18;
+
+import "ds-test/test.sol";
+import "cheats/Vm.sol";
+
+contract Contract {
+    function add(uint256 a, uint256 b) public pure returns (uint256) {
+        return a + b;
+    }
+}
+
+contract ExpectCreateTest is DSTest {
+    Vm constant vm = Vm(HEVM_ADDRESS);
+    bytes bytecode = vm.getDeployedCode("cheats/ExpectCreate.t.sol:Contract");
+
+    function testExpectCreate() public {
+        vm.expectCreate(bytecode, address(this));
```

Comment:
can we have a test with multiple / nested creates too? (would also be great for adding it in book)